js/varien/js.js

```diff
@@ -649,18 +649,6 @@ Element.addMethods({
     }
 });
 
-/*
-if (!("console" in window) || !("firebug" in console))
-{
-    var names = ["log", "debug", "info", "warn", "error", "assert", "dir", "dirxml",
-    "group", "groupEnd", "time", "timeEnd", "count", "trace", "profile", "profileEnd"];
-
-    window.console = {};
-    for (var i = 0; i < names.length; ++i)
-        window.console[names[i]] = function() {}
-}
-*/
-
 /**
  * Executes event handler on the element. Works with event handlers attached by Prototype,
  * in a browser-agnostic fashion.
```

Comment:
May I ask why this if statement was removed from code? What was the purpose?

Reply:
Are you asking why it was removed in this PR? It was commented out by Magento.

If you're asking why they commented it out... 🤷

Reply:
In magento there are certain parts commented in the code but not because they would not be functional but because they served some purposes. I asked why this part of the code was removed by the PR author since I see no connection to IE.

Reply:
`!("console" in window)` was for IE 6. It didn't have console.log (crazy, but true... that's how old it is)

Reply:
https://caniuse.com/console-basic

Reply:
Thank you for your clarification. In this case personally I no longer have any objection related to PR.